ui/page/main_page.go

```diff
@@ -511,10 +524,39 @@ func (mp *MainPage) LayoutTopBar(gtx layout.Context) layout.Dimensions {
 											})
 									}),
 									layout.Rigid(func(gtx C) D {
+										if mp.hideBalance.Changed() {
+											mp.hide = !mp.hide
+										}
```

Comment:
This is not needed, you can always get the current value from `mp.hideBalance.Value`